build.sbt

```diff
@@ -3927,6 +3927,9 @@ lazy val `engine-runner` = project
             .listFiles("*.jar")
             .map(_.getAbsolutePath()) ++
           `std-aws-polyglot-root`.listFiles("*.jar").map(_.getAbsolutePath()) ++
+          `std-microsoft-polyglot-root`
+            .listFiles("*.jar")
+            .map(_.getAbsolutePath()) ++
```

Comment:
- for now we want to keep building with `Standard.Microsoft` libraries processed
- thus 20529f8b984856ddf153b418968c6b72134de6a1 returns it back